### DISCUSSION TITLE
Which one to choose for VSCode extension between mdx-js/vscode-mdx and silvenon/vscode-mdx?

### DISCUSSION THREAD

**Discussion by GorvGoyl** [mdx-js/mdx#1373] [Q&A] [answered]
I see there are 2 repos https://github.com/mdx-js/vscode-mdx and https://github.com/silvenon/vscode-mdx which provides MDX support for VSCode. Could someone please elaborate the difference between them?

**Comment by ChristianMurphy** [comment, marked as answer]
Both offer syntax highlighting for MDX, both are created and maintained by MDX contributors/maintainers.
https://github.com/mdx-js/vscode-mdx is maintained in the official MDX org organization.
https://github.com/silvenon/vscode-mdx is maintained on a personal repository.
@JounQin and @silvenon may be able to comment on differences between the extensions.

**Reply by silvenon** [reply]
At the moment you should decide for yourself which one works better for you.

I haven't been maintaining mine for a long time, although I'm still around for discussions in issues and PRs. My only active plan for it was to add syntax highlighting at the time when the other MDX extension didn't exist, for anything more it seems like I'd need to learn much more about VS Code extension development, which I don't plan to for the foreseeable future, but it's not ruled out either.

I actually wanted to see if we can somehow merge the two projects together 🤞 but haven't got around to seriously initiating this.
